src/spirc.rs

```diff
@@ -246,7 +295,9 @@ impl SpircTask {
 
         if frame.get_ident() == self.ident ||
            (frame.get_recipient().len() > 0 && !frame.get_recipient().contains(&self.ident)) {
-            return;
+            if frame.get_typ() == MessageType::kMessageTypeNotify {
```

Comment:
Not too comfortable with that, it'll start reacting to messages targeted at other devices.

I don't think we should send messages to ourselves. I suggest making `handle_command` do `if active { self.handle_play(); } else { broadcast_message(kMessageTypePlay); }`